### PR TITLE
Added get_stdout and get_resources_usage to task class

### PR DESCRIPTION
**Pull request by david451lima**
Tasks objects now have a get_stdout and get_resources_usage that returns the respective information.
The functions simply call the server side endpoints defined in the PR [#367](https://github.com/inductiva/inductiva-web-api/pull/367)

### CODE REVIEW

File: inductiva/tasks/task.py [hunk 1]
```diff
@@ -411,3 +411,51 @@ def get_machine_type(self) -> Optional[str]:
...
+
+        for line in api_response.body:
+            print(line)
+
+        return api_response.body
+
+    def get_resources_usage(self, n_lines: int = 10):
+        """Returns tail of resources_usage.txt file for current task
+
+        Calls the get_file_tail function, specifying the path of the
```
Comment (IvanPombo) [inline inductiva/tasks/task.py:439]: I guess now is "get_resources_tail"?

Comment (IvanPombo) [reply inductiva/tasks/task.py:439]: The general docstring should be updated.

**Comment by david451lima** [comment]
> Can you show the print examples? After, confirming that, it looks good to merge!

```python

scenario = inductiva.molecules.ProteinSolvation(pdb_file)

task = scenario.simulate(simulation_time_ns=2.01,
                             n_steps_min = 100, run_async = True)#,
                            # machine_group = machine)

sleep(145)
print(task.get_stdout())
´´´

yields

```
INFO:absl:Task ID: 1698073361633479796
INFO:absl:Creating input archive...
INFO:absl:Input archive created.
INFO:absl:Input archive size: 3.20 KiB
INFO:absl:Uploading input...
INFO:absl:Input archive uploaded.
INFO:absl:Task request submitted.
Calculating fourier grid dimensions for X Y Z
Using a fourier grid of 24x24x24, spacing 0.117 0.117 0.117

Estimate for the relative computational load of the PME mesh part: 0.17

This run will generate roughly 0 Mb of data

 -------
# COMMAND: gmx mdrun -deffnm em -v yes 

('Calculating fourier grid dimensions for X Y Z', 'Using a fourier grid of 24x24x24, spacing 0.117 0.117 0.117', '', 'Estimate for the relative computational load of the PME mesh part: 0.17', '', 'This run will generate roughly 0 Mb of data', '', ' -------', '# COMMAND: gmx mdrun -deffnm em -v yes ', '')
´´´

**Comment by IvanPombo** [comment]
Thanks @david451lima!! So a couple of small comments regarding this:
- Make a small break between the usual logs and these gets;
- Add initial messages stating what we are printing to the console;
- Add a header describing the Resources.

Why is that last line on the resources like that?

File: inductiva/tasks/task.py [hunk 2]
```diff
@@ -411,3 +412,69 @@ def get_machine_type(self) -> Optional[str]:
...
+            query_params={
+                "n_lines": n_lines,
+            },
+            stream=False,
+            skip_deserialization=False,
+        )
+
+        if verbose:
+            logging.info(
+                "Yielding the current resource usage of the simulation:")
```
Comment (IvanPombo) [inline inductiva/tasks/task.py:466]: Suggestion: "Current resource usage"

File: inductiva/tasks/task.py [hunk 3]
```diff
@@ -411,3 +412,69 @@ def get_machine_type(self) -> Optional[str]:
...
+            path_params=self._get_path_params(),
+            query_params={
+                "n_lines": n_lines,
+            },
+            stream=False,
+            skip_deserialization=False,
+        )
+
+        if verbose:
+            logging.info("Yielding the output from stdout:")
```
Comment (IvanPombo) [inline inductiva/tasks/task.py:435]: Simulation stdout:

**Review by IvanPombo — APPROVED** [review]
LGTM!

**Pull request merged**